### PR TITLE
Add profiles bus values

### PR DESCRIPTION
**Pull request by seanwiseman**

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
For https://github.com/elifesciences/profiles/pull/52.

File: salt/profiles/config/srv-profiles-app.cfg [hunk 1]
```diff
@@ -12,3 +12,9 @@ client_secret = {{ pillar.profiles.orcid.client_secret }}
 [logging]
 path = /srv/profiles/var/logs
 level = {{ pillar.profiles.logging.level }}
+
+[bus]
+region = {{ pillar.profiles.sns.region }}
+subscriber = {{ pillar.profiles.sns.subscriber }}
+name = {{ pillar.profiles.sns.name }}
+env = {{ pillar.elife.env}}
```
Comment (giorgiosironi) [inline salt/profiles/config/srv-profiles-app.cfg]: missing space before `}}`

Comment (seanwiseman) [reply salt/profiles/config/srv-profiles-app.cfg]: 👍🏻

File: salt/pillar/profiles.sls [hunk 2]
```diff
@@ -22,6 +22,10 @@ profiles:
         level: DEBUG
     orcid_dummy:
         pinned_revision_file: /srv/profiles/orcid-dummy.sha1
+    sns:
+        name: bus-profiles
```
Comment (giorgiosironi) [inline salt/pillar/profiles.sls]: this name (when tied together with the `--$env` suffix) is the correct one, but you should correct https://github.com/elifesciences/profiles-formula/blob/master/salt/profiles/init.sls#L136 to use this, that name was a placeholder

Comment (seanwiseman) [reply salt/pillar/profiles.sls]: OK have updated the name.

File: salt/pillar/profiles.sls [hunk 3]
```diff
@@ -22,6 +22,10 @@ profiles:
         level: DEBUG
     orcid_dummy:
         pinned_revision_file: /srv/profiles/orcid-dummy.sha1
+    sns:
+        name: profiles
```
Comment (giorgiosironi) [inline salt/pillar/profiles.sls]: I meant the other way around: your name was the correct one (compare https://github.com/elifesciences/builder/blob/master/projects/elife.yaml#L994) and the `aws sns` command that creates a topic for testingneeds to use this pillar value

**Comment by giorgiosironi** [comment]
Systemd problems in starting the service. Taking a look.

**Comment by giorgiosironi** [comment]
Reproduces, not in Vagrant but only AWS.

**Comment by giorgiosironi** [comment]
Not very helpful:
```
● uwsgi-profiles.service - "uwsgi profiles"
   Loaded: loaded (/lib/systemd/system/uwsgi-profiles.service; disabled; vendor preset: enabled)
   Active: inactive (dead)

Nov 08 12:39:27 profiles-formula-manual-test--profiles.elifesciences.org systemd[1]: Starting "uwsgi profiles"...
Nov 08 12:39:27 profiles-formula-manual-test--profiles.elifesciences.org newrelic-admin[1079]: [uWSGI] getting INI configuration from /srv/
Nov 08 12:39:28 profiles-formula-manual-test--profiles.elifesciences.org systemd[1]: uwsgi-profiles.service: Main process exited, code=exit
Nov 08 12:39:28 profiles-formula-manual-test--profiles.elifesciences.org systemd[1]: Failed to start "uwsgi profiles".
Nov 08 12:39:28 profiles-formula-manual-test--profiles.elifesciences.org systemd[1]: uwsgi-profiles.service: Unit entered failed state.
Nov 08 12:39:28 profiles-formula-manual-test--profiles.elifesciences.org systemd[1]: uwsgi-profiles.service: Failed with result 'exit-code'
Nov 08 12:39:28 profiles-formula-manual-test--profiles.elifesciences.org systemd[1]: uwsgi-profiles.service: Service hold-off time over, sc
Nov 08 12:39:28 profiles-formula-manual-test--profiles.elifesciences.org systemd[1]: Stopped "uwsgi profiles".
Nov 08 12:39:28 profiles-formula-manual-test--profiles.elifesciences.org systemd[1]: uwsgi-profiles.service: Start request repeated too qui
Nov 08 12:39:28 profiles-formula-manual-test--profiles.elifesciences.org systemd[1]: Failed to start "uwsgi profiles".
```

**Comment by giorgiosironi** [comment]
```
Traceback (most recent call last):
  File "/srv/profiles/venv/lib/python3.5/site.py", line 703, in <module>
    main()
  File "/srv/profiles/venv/lib/python3.5/site.py", line 694, in main
    execsitecustomize()
  File "/srv/profiles/venv/lib/python3.5/site.py", line 548, in execsitecustomize
    import sitecustomize
  File "/srv/profiles/venv/lib/python3.5/site-packages/newrelic/bootstrap/sitecustomize.py", line 136, in <module>
    newrelic.config.initialize(config_file, environment)
  File "/srv/profiles/venv/lib/python3.5/site-packages/newrelic/config.py", line 2647, in initialize
    log_file, log_level)
  File "/srv/profiles/venv/lib/python3.5/site-packages/newrelic/config.py", line 889, in _load_configuration
    'Unable to open configuration file %s.' % config_file)
newrelic.api.exceptions.ConfigurationError: Unable to open configuration file /srv/profiles/newrelic.ini.
```
boils down to newrelic-python not being executed soon enough to provide newrelic.ini. Similar problem as for other projects.

**Comment by giorgiosironi** [comment]
Now green, let's merge.

**Pull request merged**